transports/stdio/runtime/src/main/java/io/quarkiverse/mcp/server/stdio/runtime/StdioMcpMessageHandler.java

```diff
@@ -57,7 +57,7 @@ protected StdioMcpMessageHandler(McpRuntimeConfig config, ConnectionManager conn
         this.vertx = vertx;
     }
 
-    void initialize(PrintStream stdout, McpRuntimeConfig config) {
+    public void initialize(PrintStream stdout, McpRuntimeConfig config) {
```

Comment:
Since we're making this public and the `StdioMcpMessageHandler` is a CDI bean, could you pls add the `private final AtomicBoolean initialized = new AtomicBoolean()` field and only initialize if `initialize.compareAndSet(false, true)` returns `true`?

Reply:
Done.

Reply:
Cool, thanks!